R/teal.R

```diff
@@ -1,10 +1,10 @@
-#' teal: Interactive Exploration of Clinical Trials Data
+#' `teal`: Interactive exploration of clinical trials data
```

Comment:
I think it can still say "clinical trials data" as we use the same term in the `DESCRIPTION`. Technically people can use data from multiple clinical trials so I prefer having the plural here.

Reply:
ok